code/_onclick/item_attack.dm

```diff
@@ -17,6 +17,10 @@
 	user.changeNext_move(CLICK_CD_MELEE)
 	if(stat == DEAD && !isnull(butcher_results)) //can we butcher it?
 		if(istype(I, /obj/item/weapon/kitchen/knife))
+			if(istype(I, /obj/item/weapon/kitchen/knife/hunting))
```

Comment:
I initially made it so you could use any knife, but the second after the head was cut off, the butchering process started, and I found no other way to remedy this.